torch/csrc/jit/codegen/cuda/test/test_gpu.cpp

```diff
@@ -13795,7 +13796,8 @@ TEST_F(NVFuserTest, FusionSimpleVectorizeUnroll_CUDA) {
   tv3->reorder({{4, 2}});
   // [bidx, unswitch, vectorize{2}, unroll{2}, tidx]
 
-  TransformPropagator(tv3).run();
+  TransformPropagator propagator(tv3);
+  MaxRootDomainInfoSpanningTree(tv3).traverse(&propagator);
```

Comment:
Would be nice to have a concise shortcut. Average users won't be concerned about the C++ interface, but it is still an interface for experts.

Reply:
I like the idea of having a shortcut in scheduler utils. But in unit tests, I am considering using `TransformPropagatorWithCheck` to replace all the `TransformPropagatorWith` https://github.com/csarofeen/pytorch/pull/1791#discussion_r911739913, so probably in the unit tests, we will not have much chance to use that shortcut.